src/Microsoft.DotNet.Interactive.Tests/LanguageKernelTestBase.cs

```diff
@@ -74,33 +75,40 @@ public void Dispose()
             _lockReleaser.Dispose();
         }
 
+        protected CompositeKernel CreateFullKernel(Language defaultKernelLanguage = Language.CSharp)
```

Comment:
```suggestion
        protected CompositeKernel CreateCompositeKernel(Language defaultKernelLanguage = Language.CSharp)
```

It's good to avoid introducing alternative terms for an already-named concept.